wp-content/themes/missilethreat/functions.php

```diff
@@ -247,6 +247,15 @@ function missilethreat_register_scripts() {
 	// Font Awesome
 	wp_enqueue_script('missiledefense-font-awesome-js', 'https://use.fontawesome.com/08b1a76eab.js');
 
+	if ( is_singular( array( 'defsys', 'missile' ) )  || is_single() ) {
```

Comment:
Did you test this conditional with just `is_single()`? It's my understanding that `is_singular()` would be redundant to use here because `is_single()` includes it. But I could be wrong!